custom_components/solarfocus/sensor.py

```diff
@@ -313,11 +314,12 @@ def native_value(self):
         key="mode",
         icon="mdi:format-list-bulleted",
         device_class=SensorDeviceClass.ENUM,
-        options=list(range(0, 3)),
+        options=list(range(0, 5)),
     ),
     SensorEntityDescription(
         key="single_charge",
         icon="mdi:pump",
+        device_class="solarfocus__bocharge"
```

Comment:
no required anymore as we are using `translation_key`

Reply:
so just remove is sufficient!?

Reply:
to reference in `strings.json` use `bo_single_charge`

Reply:
But thats just a button -  so what then the content?
![grafik](https://user-images.githubusercontent.com/72338417/232335656-33f49509-03db-4ebf-bd5f-74bd34a17690.png)